source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,346 @@ Sorts Builders
    :class: singlecol
 
 .. _sorts-builders:
+
+Overview
+--------
+
+In this guide, we show you how to use **builders** to specify
+**sort criteria** for your queries.
+
+Sort criteria are the rules MongoDB uses to sort your data. Some examples of sort criteria are:
+
+* Smallest number to largest number
+* Earliest time of day to latest time of day
+* Alphabetical order by first name 
+
+Builders are classes provided by the MongoDB Java driver that allow you to use
+the builder pattern to interact with your MongoDB instance.
+The builder pattern allows you to create complex
+immutable objects separately from their representation. The
+:doc:`builders </fundamentals/builders/>`
+provided by the MongoDB Java driver can improve the readability of your code.
+
+You should read this guide if you would like to:
+
+* Use the builder pattern to specify sort criteria for your queries.
+* Learn how to perform ascending sorts, descending sorts, and combining sorts.
+* Sort on the text score of :manual:`$text </reference/operator/query/text/>` 
+  queries.
+
+If you are new to sorting in the
```

Comment:
How about "If you want to learn the fundamentals of sorting using the MongoDB Java driver"? We should avoid language that might be discouraging such as "you're new", "simple", etc.

Reply:
If you change it to my suggestion, make sure the rest of the paragraph makes sense and doesn't repeat itself.

Reply:
Thanks for the note. Good catch, didn't notice how this could be discouraging but it definitely can be. Fix: https://github.com/mongodb/docs-java/pull/59/commits/dea12b0e7a2f86cc0171f6a7a56eac58b249fcb9

Going forward, I will try to get a better sense of when I am using discouraging language in my writing.